android/Tests/KeyboardHarness/app/src/main/assets/cloud/platformtest.js

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
This file needs to be renamed to `platformtest-1.0.js` because legacy. 😢  It took me a while to figure out why the harness couldn't switch to the keyboard when I realized the version was blank.

Reply:
@mcdurdin wrote "haven't run the tests" so we should have known it wouldn't work 😉